### PR TITLE
Add initialize eslint configuration

### PR DESCRIPTION
**Pull request by alexander-schranz**
Inspired by https://github.com/netvlies/eslint-config-nvs and https://github.com/vicompany/eslint-config-vi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
-# eslint-config-ma
\ No newline at end of file
+# eslint-config-ma
+
+![CircleCi](https://circleci.com/gh/massiveart/eslint-config-ma/tree/master.png)
+
+Stylelint shareable config used by MassiveArt.
```
Comment (c00n84) [inline README.md]: @alexander-schranz please write "MASSIVE ART"

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/massiveart/eslint-config-ma.git"
+  },
+  "keywords": [
+    "eslint",
+    "eslintconfig",
+    "massiveart"
+  ],
+  "author": "MassiveArt Webservices GmbH",
```
Comment (c00n84) [inline package.json]: @alexander-schranz MASSIVE ART WebServices GmbH

File: package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "eslint-config-ma",
+  "version": "1.0.0",
+  "description": "ESLint shareable config used by MassiveArt",
```
Comment (c00n84) [inline package.json]: @alexander-schranz MASSIVE ART

**Comment by c00n84** [comment]
Else ACK

**Comment by thomasduenser** [comment]
@alexander-schranz ACK

**Pull request merged**